src/tests/ftest/util/command_utils_base.py

```diff
@@ -477,20 +477,6 @@ def __init__(self, namespace, title):
         """
         super(TransportCredentials, self).__init__(namespace, None, title)
 
-        # Transport credential parameters:
-        #   - allow_insecure: false|true
-        #       Specify 'false' to bypass loading certificates and use insecure
-        #       communications channels
-        #
-        #   - ca_cert: <file>, e.g. ".daos/daosCA.crt"
-        #       Custom CA Root certificate for generated certs
-        #
-        #   - cert: <file>, e.g. ".daos/daos_agent.crt"
-        #       Agent certificate for use in TLS handshakes
-        #
-        #   - key: <file>, e.g. ".daos/daos_agent.key"
-        #       Key portion of Server Certificate
-        #
         self.allow_insecure = BasicParameter(True, True)
```

Comment:
Should this be changed to **secure by default**?

Reply:
same as above.

Reply:
Ditto.